models/model.js

```diff
@@ -26,7 +25,22 @@ exports.selectArticles = (sort_by = "created_at", order = "DESC") => {
   queryString += `ORDER BY ${sort_by} ${order};`;
 
   return db.query(queryString).then(({ rows }) => {
-    console.log(rows);
     return rows;
   });
 };
+
+exports.selectArticlesById = (id) => {
+  let queryString = ` 
+  SELECT * FROM articles WHERE article_id = $1`;
+
+  return db.query(queryString, [id]).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "article not found" });
+    }
+    if (!rows) {
+      return Promise.reject({ status: 400, msg: "invalid id" });
```

Comment:
does this need to be here, I would imagine that if you give an invalid Id this promise would reject? 
If that's the case I would make sure to remove this as its code that is not needed